src/foam/lib/json/OutputJSON.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+foam.INTERFACE({
```

Comment:
Insert blank line.
Did you remove src/foam/lib/json/OutputJSON.java?

Reply:
no

Reply:
You should.

Reply:
already done